tests/test_etherscan.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+import pytest
+from requests import Response
+
+
+@pytest.fixture
+def etherscan_abi_response(mocker):
+    response = mocker.MagicMock(spec=Response)
+    f = open("api.json")
+    response.json.return_value = json.load(f)
+    f.close()
+    return response
+
+
+def test(mocker):
+    get_patch = mocker.patch("ape_etherscan.explorer.requests")
+    get_patch.get.return_value = etherscan_abi_response
```

Comment:
The test is only setting up right now, it is not asserting anything.
We need to create an instance of the `Etherscan` class and call the method and make sure we get the expected return value

Reply:
I'm glad you checked on this, thank you 👍

Reply:
would this be something like `networks.ethereum.mainnet.explorer.get_contract_type(etherscan_abi_response)`?

Reply:
Not quite!

So, when we call `get_contract_type()`, we are expecting to get the `etherscan_abi_response` as the response.

Looks like we just need to give it an address (but it does not have to be a real address or anything because we are just going to get the mock)

Reply:
```    
actual = networks.ethereum.mainnet.explorer.get_contract_type("TEST")
assert actual.contractName == "BoredApeYachtClub"
```

Reply:
flake8 doesn't like me using networks. i think i have to import it.

Reply:
`from ape import networks`